clang/include/clang/AST/Decl.h

```diff
@@ -5179,6 +5179,8 @@ class HLSLRootSignatureDecl final
                                     llvm::hlsl::rootsig::RootElement> {
   friend TrailingObjects;
 
+  llvm::dxil::RootSignatureVersion RootSigVer;
```

Comment:
This is a member of a HLSLRootSignatureDecl - it can probably just be called Version.